Contribution,md

```diff
@@ -0,0 +1,60 @@
+
+We welcome contributions in the form of code, documentation, bug reports, feature requests, and more. By contributing to this project, you're helping make game development using Unreal Engine more accessible and efficient.
+
+Make sure you read the full documentation available [here](https://hfjooste.github.io/UltimateStarterKit/). The Markdown documentation is also included in the repo under the docs directory you can also refer to it offline.
+
+## How to Contribute
+
+To contribute to the Ultimate Starter Kit, please follow these guidelines:
+
+1. **Fork the Repository**: Click the "Fork" button at the top right of this repository to create your own fork.
+
+2. **Clone Your Fork**: Clone your fork of the repository to your local development environment using the following command:
+   ```bash
+
+   git clone https://github.com/your-username/Ultimate-Starter-Kit.git
+   ```
+3.**Create a New Branch:** Before making any changes, create a new branch for your contribution. Please choose a descriptive name for your branch.
+```bash
+git checkout -b feature/new-feature
+```
+4.**Make Changes:** Make your desired changes or contributions to the project.
+
+5.**Testing:** If you are introducing new features or making changes, ensure that you test your changes to confirm they work as intended. We encourage unit tests, integration tests, or any other relevant testing methodology.
+
+6.**Documentation:** If your changes affect user documentation, be sure to update or create the necessary documentation to explain the changes.
+
+7.**Commit Your Changes:** Commit your changes with a clear and concise message that explains the purpose of the change.
+
+
+```bash
+git commit -m "Add feature: [Your Feature Name]"
+```
+8.**Push Your Changes:** Push your changes to your fork on GitHub.
+```bash
+git push origin feature/new-feature
+```
+9.**Open a Pull Request:** Go to the original repository, and on the main page, you should see a "Compare & pull request" button. Click it and provide a clear and detailed description of your changes. Make sure to reference any related issues or pull requests.
+
+10.**Review and Discussion:** Be prepared for discussions and potential code review. Address any comments or feedback received during the review process.
+
+11.**Merge:** Once your pull request is approved, it will be merged into the main repository. Congratulations on your contribution!
+
+## Code of Conduct 
+
+We expect all contributors to follow our Code of Conduct. Please be respectful and considerate when interacting with other community members.
+
+## Reporting Issues
+
+If you encounter bugs, issues, or have feature requests, please open a new issue on the GitHub repository. Provide detailed information about the problem you're experiencing or the feature you'd like to see.Be sure you are using Unreal Engine 4.27 and newer versions.
```

Comment:
Please add a space between `see.` and `Be`. Also I think it sounds better to replace `Be` with `Make`